### ISSUE TITLE
Crash without internet

### ISSUE THREAD

**Issue by SomeAspy** [ytmdesktop/ytmdesktop#907]
### Consent

- [x] I verified that there is no open issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

The app will crash with a fatal JS error when starting without internet.
However, once connected to internet and the error window is closed, it will recover.
This bug seems to have been created in the latest Nightly release. (The app does not reflect this, and shows 1.13.0)
![rbh9](https://user-images.githubusercontent.com/33640860/153588592-b989bf16-9db4-4f5f-874d-ff96e1228431.png)

### Expected Behavior

The app should open without firing a fatal JS error, and bring the user to the "looks like your not connected" page.

### Steps To Reproduce

1. Disconnect internet
2. Start Application
3. Observe fatal JS error

### YTMDesktop

Nightly Nov 2021 1.14.1 (App shows 1.13.0)

### OS

Windows

### OS Version

11

### Arch

x64

### Installation way

.exe file from github releases

### Anything else?

I was unable to locate where logs for the app are stored.

**Comment by SomeAspy** [comment]
This seems to be fixed as of [e469e2c](https://github.com/ytmdesktop/ytmdesktop/commit/e469e2c1a0218f2ba541747145461029b22141d1)
So once the development branch is merged into stable, this issue will be resolved

**Comment by SomeAspy** [comment]
Closed due to rewrite
